app/static/openapi.yaml

```diff
@@ -722,6 +741,96 @@ components:
             message:
               description: Error message
               type: string
+    Error_list:
+      type: object
```

Comment:
`Error_list` is an `object`? why is it not an `array`?

Reply:
I just kind of ran out of names that I could come up with. This object is the exact same as `Error`, except instead of `Error_data` as its data, it has `Error_list_data`. If there is a way in yaml to use the same error object but specify a different `data`, I'd love to use that, but I'm unaware of a way to do that.

That being said, I'm open to other options about what this should be named. `Error_list_object`?

Reply:
I think there are a few problems with our naming conventions in this file, and I think that existed before I got to it. I'm going to call that out of scope for this PR, and open a separate issue in the repo for it

Reply:
https://github.com/OperationCode/resources_api/issues/259